lib/init/config-initializer.js

```diff
@@ -164,29 +164,17 @@ function processAnswers(answers) {
         config.env[env] = true;
     });
 
+    if (answers.typescript) {
+        config.extends.push("plugin:@typescript-eslint/recommended");
+    }
+
     // add in library information
     if (answers.framework === "react") {
-        config.parserOptions.ecmaFeatures = {
-            jsx: true
-        };
-        config.plugins = ["react"];
         config.extends.push("plugin:react/recommended");
     } else if (answers.framework === "vue") {
-        config.plugins = ["vue"];
-        config.extends.push("plugin:vue/essential");
-    }
-
-    if (answers.typescript) {
-        if (answers.framework === "vue") {
+        config.extends.push("plugin:vue/vue3-recommended");
```

Comment:
emm, given vue3 is the latest version, I'm not a big fan to make it too complicated - it could be a good default for most users, but they can modify the config if they want to custom.

Reply:
ok~